addons/l10n_sa_edi_pos/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'Saudi Arabia - E-invoicing (Simplified)',
+    'icon': '/l10n_sa/static/description/icon.png',
+    'version': '0.1',
+    'depends': [
+        'l10n_sa_pos',
+        'l10n_sa_edi',
+    ],
+    'author': 'Odoo',
+    'summary': """
+        ZATCA E-Invoicing, support for Simplified Invoices
```

Comment:
Do not we have support for simplified invoices in the main module already?  (in other words, should not we say support for PoS)